client/post-editor/editor-drawer/index.jsx

```diff
@@ -321,7 +324,9 @@ const EditorDrawer = React.createClass( {
 					onTrashingPost={ this.props.onTrashingPost }
 					onPrivatePublish={ this.props.onPrivatePublish }
 					setPostDate={ this.props.setPostDate }
-					site={ this.props.site }>
+					site={ this.props.site }
+					status={ postStatus }
+				>
```

Comment:
Minor: In moving the opening `>`, we could have also made this self-closing `/>`.